tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -187,3 +196,33 @@ func createDownstreamCluster(client *rancher.Client, clusterType string) (*manag
 
 	return clusterObject, steveObject, testClusterConfig, nil
 }
+
+func getErrorMessage(err, startText, endText string) (actualError string) {
```

Comment:
Can we add this to extension since this is generic and can be used by other packages too?